dags/operators/gcp_container_operator.py

```diff
@@ -80,57 +111,13 @@ def _set_env_from_extras(self, extras):
         if not key_path and not keyfile_json_str:
             self.log.info('Using gcloud with application default credentials.')
         elif key_path:
-            os.environ[G_APP_CRED] = key_path
-            os.environ[GCLOUD_APP_CRED] = key_path
+            os.environ[CREDENTIALS] = key_path
+            return None
```

Comment:
we dont hit this code path, but i left this line here to preserve diffs with upstream